### ISSUE TITLE
Inspire should not be able to target yourself

### ISSUE THREAD

**Issue by itamarcu** [mlenser/kryx-rpg-issues#177]
https://www.kryxrpg.com/themes/maneuvers/inspire

Also, note: this maneuver might be too strong compared to [Precision](https://www.kryxrpg.com/themes/maneuvers/precision) (or Precision might be too weak). It adds a bonus to saving throws too, it lets you target allies, and you can enhance it to help multiple allies simultaneously.

Do note, when balancing these two maneuvers, that the bonus is applied to *all* attack rolls/saves, and not just one.

**Comment by mlenser** [comment]
Inspire is saving throws or attack, never both. The added versatility is nice, but not deal-breaking.

Pros of Inspire:
- Can use at range
- Can boost saving throws or attacks
- Can target multiple creatures

Cons of Inspire:
- Your ally might have used its reaction
- Your ally is not your damage boost bot

The teamplay required here outweighs the issue. If you have players picked Courage to self-buff eachother because "Precision is weak" then they have an optimization problem and have taxed themselves 1 theme to get what they're after. Fair, I'd call it.

**Comment by itamarcu** [comment]
> Inspire is saving throws or attack, never both

Ah! This is really unclear with the way it's currently phrased, then, IMO. Could you rephrase it somehow? "Choose saving throws or attack rolls", or something like that?

**Comment by mlenser** [comment]
`the creature has a +1 bonus to attack or saving throw rolls`
is very different from
`the creature has a +1 bonus to attack and saving throw rolls`

**Comment by itamarcu** [comment]
I'm in favor, yes!
